cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -60,11 +63,10 @@ def full_cluster(cfy, ssh_key, module_tmpdir, attributes,
 
 
 @pytest.fixture()
-def full_cluster_with_lb(cfy, ssh_key, module_tmpdir, attributes,
-                         logger):
+def cluster_with_lb(cfy, ssh_key, module_tmpdir, attributes, logger):
     for _vms in _get_hosts(cfy, ssh_key, module_tmpdir,
                            attributes, logger,
-                           broker_count=3, db_count=3, manager_count=2,
+                           broker_count=2, db_count=1, manager_count=3,
```

Comment:
Depending on what is being tested with this cluster, you might be able to drop to one broker and two managers?